server/server.go

```diff
@@ -1483,6 +1538,10 @@ func (s *Server) Shutdown() {
 	}
 	s.Noticef("Initiating Shutdown...")
 
+	if s.accResolver != nil {
```

Comment:
edit: I think a DirAccResolver embeds DirJWTStore and therefore would invoke the later's Close() function. Right?

Reply:
Yes. Because DirAccResolver is embedded directly (it's close is what is called)

Similarly, in accounts.go `type resolverDefaultsOpsImpl struct{}` is embedded by Mem and Url resolver. 
With this, they only need to overwrite what differs from these defaults